source/_cookbook/customize_polling_interval.markdown

```diff
@@ -7,8 +7,8 @@ sidebar: true
 comments: false
 sharing: true
 footer: true
-ha_category: Example configuration.yaml
----
+ha_category: Customize Defaults
+-------------------------------
```

Comment:
Those were automatically added by PHPStorm and the MultiMarkdown plugin... I didn't notice it wasn't supposed to be like that. I'll fix it.